CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
+## [2.05.0.0.0]
+
+This software update is a consensus changing release and the
+implementation of the proposed cost changes in SIP-012. This release's
+chainstate directory is compatible with chainstate directories from
+2.0.11.4.0, however, it is only compatible with chainstate directories
+before the 2.05 consensus changes activate (Bitcoin height 713,000). If
+you run a 2.00 stacks-node beyond this point, and wish to run a 2.05 node
+afterwards, you must start from a new chainstate directory.
+
+## Added
+
+- At height 713,000 a new `costs-2` contract will be launched by the
+  null Stacks address.
```

Comment:
Is there an official name for the zero-address?  The code calls it the "boot code address" for example.

Reply:
Yeah, I liked boot code address, but it's no longer boot code only :(

Not sure otherwise, happy to call it whatever.

Reply:
I think of it like a BIOS or a ROM -- it's the address where all the _pre-defined_ smart contract code lives that ships with the node.  It's true that it's mainly involved in bootup, but it's randomly accessed and called by other running code loaded later and it's allowed to receive new code as part of a system-wide upgrade.  That to me is consistent with calling it the "boot code address."

I guess I'm more anxious about the code and documentation calling it two different things than the address having a more appropriate name like "system address."

Reply:
Okay, updated to "Stacks boot address"